recipe/meta.yaml

```diff
@@ -23,12 +23,16 @@ requirements:
   run:
     - python
     - absl-py >=0.1.6
+    - astor >=0.6.0
     - enum34 >=1.1.6  # [py<34]
+    - gast >=0.2.0
```

Comment:
Currently `gast` is not packaged. Have raised a request for its addition in issue ( https://github.com/conda-forge/staged-recipes/issues/5704 ).